src/frontend/apps/calendars/src/features/calendar/components/scheduler/scheduler-theme.scss

```
@@ -261,10 +266,70 @@
+        }
+      }
+
+      &::after {
+        content: ", today";
+        position: absolute;
+        width: 1px;
+        height: 1px;
+        padding: 0;
+        margin: -1px;
+        overflow: hidden;
+        clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)